pom.xml

```diff
@@ -9,8 +9,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>19</maven.compiler.source>
-        <maven.compiler.target>19</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
lets keep version 19. and change intellij project settin
<img width="821" alt="Screenshot 2024-02-26 at 21 41 49" src="https://github.com/A-Diliyaer/Cube24/assets/65502064/2b166096-de95-44ad-89b7-2ee3cb703f52">
gs from 17 to 19.

Reply:
done